android/src/main/java/io/spokestack/RNSpokestack/RNSpokestackModule.java

```diff
@@ -78,10 +75,11 @@ public void initialize(ReadableMap config) throws Exception {
             Map<String, Object> map = config.getMap("pipeline").toHashMap();
             for (String k : map.keySet())
                 builder.setProperty(k, map.get(k));
-            Double p = Double.parseDouble(map.get("profile").toString());
-            PipelineProfiles profile = PipelineProfiles.values()[p.intValue()];
-            String namespacedProfile = "io.spokestack.spokestack.profile." + String.valueOf(profile);
-            builder.getPipelineBuilder().useProfile(namespacedProfile);
+            if (map.containsKey("profile")) {
+                Double p = Double.parseDouble(map.get("profile").toString());
```

Comment:
Could this be an int instead of a double? Would save you an extra conversion below.

Reply:
Sadly it RN interprets all numbers as `Double`

Reply:
That's probably more a JS thing than a RN thing.